src/components/component-playground.jsx

```diff
@@ -525,5 +548,40 @@ module.exports = React.createClass({
       orientation: contentNode.offsetHeight > contentNode.offsetWidth ?
                    'portrait' : 'landscape'
     });
+  },
+
+  _getFilteredFixtures() {
+    var components = this.props.components;
+
+    // Get components which match search text by one or more fixture names.
+    var matchedByFixtureName = _.pick(
+        _.mapValues(components, function(fixturesObject) {
+          return _.assign({}, fixturesObject, {
+          // Only select fixtures matching search text.
+            'fixtures': _.pick(
+                fixturesObject.fixtures,
+                function(fixtureProps, fixtureName) {
+                  return fixtureName.indexOf(this.state.searchText) !== -1;
+                }.bind(this)
+            )
+          });
+        }.bind(this)),
+        function(fixturesObject) {
+          return !_.isEmpty(fixturesObject.fixtures);
+        }
+    );
+
+    // Get components which match search text by component name and the
+    // currently selected one.
+    var matchedByComponentName = _.pick(
+        components,
```

Comment:
This can also stay on the line above. Imo it makes it easier to red where are you picking from.